components/x11/redshift/Makefile

```diff
@@ -32,11 +32,9 @@ COMPONENT_ARCHIVE_URL=	https://github.com/jonls/redshift/releases/download/v$(CO
 COMPONENT_LICENSE=	GPL-3
 COMPONENT_LICENSE_FILE=	COPYING
 
```

Comment:
TEST_TARGET=            $(NO_TESTS)
seems to be missing here.

Reply:
Added in 101b6a140dd59219a7f56526d339b1521fe53cba.